modules/configuring-aws-iam-for-cross-account-access.adoc

```diff
@@ -0,0 +1,115 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-19
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-aws-iam-for-cross-account-access_{context}"]
+= Configuring AWS IAM for cross-account access
+
+You can configure {aws-short} Identity Access Management (IAM) for cross-account access.
+
+.Prerequisites
+* You have administrator permissions.
+* You have created the target role and the target role has permissions to send data to Cloudwatch or S3 outputs.
+* You have permissions to update the policy for the initial and the target role.
+
+.Procedure
+
+. Create an initial account in your cluster account to assume the target role with the required permissions.
++
+The initial account can either be a user or an IAM role.
+This account assumes the target role.
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "arn:aws:iam::<target-account>:role/target-logging-role"
+        }
+    ]
+}
+----
+
+. In your target role, create a trust policy and specify the initial cluster role, and optionally an external id.
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam:::<role_or_user>/<role-name_or_user-name>"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "<your-unique-external-id>"
+                }
+            }
+        }
+    ]
+}
+----
++
+Set the `Principal.AWS` field as follows:
+
+** If the initial account is a user, use the format:
++
+`arn:aws:iam::<cluster-account>:user/<user-name>`
+
+** If the initial account is an IAM role, use the format:
++
+`arn:aws:iam::<aws-account-id>:role/<role-name>`
+
+. Configure CloudWatch or S3 permissions in the target role depending on whether you want to forward logs to the CloudWatch or S3 output:
++
+.CloudWatch permissions
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream", 
+                "logs:PutLogEvents",
+                "logs:DescribeLogGroups",
+                "logs:DescribeLogStreams"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+---- 
++
+.S3 permissions
+[source,json]
+----
+{   
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+        "Sid": "AllowRoleToPutObjects",
+        "Effect": "Allow",
+        "Principal": {
+            "AWS": "${assumerole_arn}"
+        },
+        "Action": [
+            "s3:PutObject",
+            "s3:PutObjectAcl"
+        ],
+        "Resource": [
+            "arn:${aws_partition}:s3:::${s3_bucket}",
```

Comment:
`Shall we use "arn:<aws_partition>:s3:::<s3_bucket>" instead of  "arn:${aws_partition}:s3:::${s3_bucket}"?`